lib/src/local_backend.rs

```diff
@@ -161,9 +161,8 @@ impl Backend for LocalBackend {
 
     fn read_symlink(&self, _path: &RepoPath, id: &SymlinkId) -> Result<String, BackendError> {
         let path = self.symlink_path(id);
-        let mut file = File::open(path).map_err(|err| map_not_found_err(err, id))?;
-        let mut target = String::new();
-        file.read_to_string(&mut target).unwrap();
+        let target_bytes = fs::read(path).map_err(|err| map_not_found_err(err, id))?;
+        let target = String::from_utf8(target_bytes).unwrap();
```

Comment:
nit: `fs::read_to_string()` can be used instead of `.unwrap()`
`map_not_found_err()` can determine error variant by `err.kind()`.

Reply:
Nice, I didn't even realize/remember there was a `fs::read_to_string()`. Fixed.